### PR TITLE
updated tealium tags

### PR DESCRIPTION
**Pull request by bwang-icf**
**JIRA Ticket:**
[BB2-3385](https://jira.cms.gov/browse/BB2-3385)

### What Does This PR Do?
Changing out tealium tags from  tags.tiqcdn.com to  tealium-tags.cms.gov. This was done by changing the config files and adding them as references

### What Should Reviewers Watch For?
Any missing areas where the old/improper tealium tag is references

### Validation
Deploy first to lower environments like test and then sandbox and see if traffic will appear with the blast team. Once that is confirmed, push to production and monitor.

### What Security Implications Does This PR Have?

Please indicate if this PR does any of the following:

* Adds any new software dependencies
* Modifies any security controls
* Adds new transmission or storage of data (possibly yes? changing to a different url maybe counts as new transmission?)
* Any other changes that could possibly affect security?

- [?] Yes, one or more of the above security implications apply. This PR must not be merged without the ISSO or team security engineer's approval.

### CODE REVIEW

File: _config-prod.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 tealium_env: "prod"
+tealium_url: "https://tealium-tags.cms.gov/cms-bluebutton/prod/utag.sync.js"
```
Comment (jimmyfagan) [inline _config-prod.yml:2]: To ensure that this isn't changed on production until it has been verified in lower environemtns, we should temporarily keep this as the previous value, we can update it to the new prod value after we've gotten confirmation that it's working.

File: _includes/bluebutton-ig/header.html [hunk 2]
```diff
@@ -27,7 +27,7 @@
   <script>
   window.tealiumEnvironment = "prod";
   </script>
-  <script src="//tags.tiqcdn.com/utag/cmsgov/cms-bluebutton/prod/utag.sync.js"></script>
+<script src="{{ site.tealium_url }}" type="text/javascript" async></script>
```
Comment (jimmyfagan) [inline _includes/bluebutton-ig/header.html:30]: Based on https://jira.cms.gov/browse/BLSTANALYT-9403 and https://cmsgov.slack.com/archives/CHDF24KMW/p1726764504253959?thread_ts=1726762257.658939&cid=CHDF24KMW, I don't think we need the `type` or `async` here. Hopefully find/replace makes this easy enough to update!

File: _config-staging.yml [hunk 3]
```diff
@@ -1,4 +1,5 @@
 tealium_env: "staging"
+tealium_url: "https://tealium-tags.cms.gov/cms-bluebutton/qa/utag.sync.js"
```
Comment (jimmyfagan) [inline _config-staging.yml:2]: I think we're okay to update this, but if we want to be really granular, we could also keep this as the previous value until we've verified test. I lean towards keeping the new value here and just being intentional about the timing of actually deploying this to staging, but could go either way.

**Review by jimmyfagan — COMMENTED** [review]
Looks good, just a few comments on some small edits I think we need. Also I still need to check if this works, we might need to redeploy to the test static site to be able to look for the updated script there.

File: _config-prod.yml (hunk 1 above)
Comment (bwang-icf) [reply _config-prod.yml:2]: Makes sense. I'll change that now

File: _config-prod.yml [hunk 4]
```diff
@@ -1,4 +1,9 @@
 tealium_env: "prod"
+##temporarily commenting out tealium_url with updated link and will later cut over
+#tealium_url: "https://tealium-tags.cms.gov/cms-bluebutton/prod/utag.sync.js"
+
+##old url that will be removed once BLAST team confirms that things work on their end
+tealium_url: ""https://tags.tiqcdn.com/utag/cmsgov/cms-bluebutton/prod/utag.sync.js""
```
Comment (jimmyfagan) [inline _config-prod.yml:6]: Are the double quotes here intentional? Also I don't know if the distinction is impactful or not, but previously the url was `//tags.tiqcdn.com/utag/cmsgov/cms-bluebutton/prod/utag.sync.js`, without a `https:` at the start.

**Review by jimmyfagan — APPROVED** [review]
Looks good!

**Pull request merged**